### PR TITLE
Added GinkoPay Enums

### PR DESCRIPTION
**Pull request by andrezimmermann**

### CODE REVIEW

File: src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java [hunk 1]
```diff
@@ -46,5 +46,27 @@
...
-  GlobalPay
+  GlobalPay,
+
+  @SerializedName("ginko_pay")
+  GinkoPay,
+
+  @SerializedName("yucho_pay")
+  YuchoPay,
+  @SerializedName("hamapay")
+  HamaPay,
```
Comment (reibitto) [inline src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java:57]: It seems kind of random whether there's an underscore before the "pay" or not. Should we be consistent with this?

Comment (andrezimmermann) [reply src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java:57]: Yeah, please adjust as you like, I just got the ball rolling, not sure of how the automatic formatting of `Enumeratum` would generate the string representation

Comment (reibitto) [reply src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java:57]: I think they fixed that bug that caused stuff like `NCCC` becoming `n_c_c_c`.

But we're not consistent with this in the api either:

```
Alipay
Jkopay
Merpay

PayPay
RakutenPay
LinePay
```

Comment (kgston) [reply src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java:57]: No.. it depends on whether we "feel" that `pay` should be capitalized or not..

Let's go with this?
```
ゆうちょPay (yuucho_pay)
ハマpay (hama_pay)
OKI Pay (oki_pay)
YOKA Pay(福岡銀行様) (yoka_pay_fukuoka)
YOKA Pay(熊本銀行様) (yoka_pay_kumamoto)
YOKA Pay(親和銀行様) (yoka_pay_shinwa)
ほくほくpay(北海道銀行様) (hokuhoku_pay_hokkaido)
ほくほくpay(北陸銀行様) (hokuhoku_pay_hokuriku)
こいpay(広島銀行様) (koi_pay_hiroshima)
```

Comment (andrezimmermann) [reply src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java:57]: For the "serialized name", everything is fine, works for me except the `こいpay`, I think their logo and romanization is "COIPAY"

Comment (kgston) [reply src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java:57]: That depends on whether want to standardize by the app's romanization or follow strictly the japanese language romanization 🤷‍♂️ But I'm ok with `coi_pay_hiroshima` I suppose..

Comment (andrezimmermann) [reply src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java]: I've adjusted the serialized name as the proposed values. Thanks!

**Review by reibitto — APPROVED** [review]

**Comment by andrezimmermann** [comment]
Thanks, will release this soon.

**Pull request merged**